src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1070,20 +1070,26 @@ class c { void M() { this$$.ToString() } }
             End Using
         End Function
 
-        <WpfTheory, CombinatorialData>
+        <WpfTheory, CombinatorialData, WorkItem(66903, "https://github.com/dotnet/roslyn/issues/66903")>
         Public Async Function TestTypingDotAfterExistingDot(showCompletionInArgumentLists As Boolean) As Task
             ' Starting C# 8.0 two dots are considered as a DotDotToken of a Range expression.
             ' A test above (TestTypingDotBeforeExistingDot) verifies that the completion happens
             ' if we type dot before a single dot.
-            ' However, we should not have a completion if typing dot after a dot.
+            ' However, completions after DotDotToken are different and should not provide items for members
             Using state = TestStateFactory.CreateCSharpTestState(
                               <Document>
 class c { void M() { this.$$ToString() } }
                               </Document>,
                               showCompletionInArgumentLists:=showCompletionInArgumentLists)
 
                 state.SendTypeChars(".")
-                Await state.AssertNoCompletionSession()
+                Await state.AssertCompletionSession()
+                Await state.AssertCompletionItemsDoNotContainAny("M")
```

Comment:
There shouldn't be completion up here. Second dot should not trigger it. You should be able to explicitly invoke completion (like ctrl-space), but dot should not trigger it.